portfolio-project-matching-app/components/Login.js

```diff
@@ -0,0 +1,42 @@
+import Button from '../components/Button'
+import Input from '../components/Input'
+
+const Login = ({ toggleNewUser }) => {
```

Comment:
My thought was that the login/signup page would use state to toggle between the `Login.js` and `Signup.js` components.